apis/core.oam.dev/v1alpha2/core_types.go

```diff
@@ -439,6 +439,9 @@ type WorkloadStatus struct {
 	// ComponentRevisionName of current component
 	ComponentRevisionName string `json:"componentRevisionName,omitempty"`
 
+	// ObservedGeneration indicates the generation observed by the appconfig controller.
+	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
```

Comment:
why do we need to add now but not used in apply-once-only before?

Reply:
apply-once-only determines a generation change according to information saved in labels and annotation. But in apply-once-apply-force, existing obj may be nil, we cannot get information to compare, so we need observedGeneration saved in ac.status.workloads.

Reply:
good , why not explain it in the annotation

Reply:
sure, I will add more explanation in detail.